public_html/app/Helpers/Constants.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace VFramework\Helpers;
+
+define("REG_RULES",
```

Comment:
What? Delete this rules, this is global constant. You dont need global constants for validation rules.
use class constants.